### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by Xupet**
Fix knob bug for Android

### CODE REVIEW

**Comment by Xupet** [comment]
Fix error that made the knob on the Agenda that got cut because of the height of the container. Increased the height by 4pts and seems like it works better now.

File: src/agenda/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import ReservationsList from './reservation-list';
 import styleConstructor from './style';
 import { VelocityTracker } from '../input';
 
-const HEADER_HEIGHT = 104;
+const HEADER_HEIGHT = 108;
```
Comment (sibelius) [inline src/agenda/index.js]: can't this be a prop instead?

Comment (tautvilas) [reply src/agenda/index.js]: probably at lerast this should be different for different platform? If IOS is fine with 104

Comment (carlosroberto555) [reply src/agenda/index.js]: Use like this

```jsx
import { Platform } from 'react-native'

const HEADER_HEIGHT = Platform.OS === 'ios' ? 104 : 108
```

Comment (carlosroberto555) [reply src/agenda/index.js]: Use Platform api to get specific value

Comment (HaswinVidanage) [reply src/agenda/index.js]: The issue is still there for much smaller screens (tested with 3.4")
Created the following PR as a fix https://github.com/wix/react-native-calendars/pull/953

**Comment by Inbal-Tish** [comment]
@Xupet Please see PR #953

**Pull request closed without merging**